### DISCUSSION TITLE
backup registry of quay.io

### DISCUSSION THREAD

**Discussion by dongho-jung** [argoproj/argo-cd#16337] [Ideas]
Quay.io has been down for several hours, preventing me from spinning up my ArgoCD cluster. While Dockerhub hasn't been updated beyond version 2.6.15, ECR gallery is more current. However, ECR version encounters a '127 command' error, which suggests it might operate differently in my env compared to quay.io.

It would be better if we have a backup registry as an alternative.

**Comment by fmbahrt** [comment]
You could sync the images to a private registry.

**Comment by crenshaw-dev** [comment]
In my opinion, a few-hours registry outage doesn't justify the maintenance overhead of pushing to multiple registries. Production environments should be using pull-through caches anyway.
